### PR TITLE
Fixes ExtensionUtils.run_command_get_output()

### PR DESCRIPTION
**Pull request by D1v38om83r**
ExtensionUtils.run_command_get_output() now decodes the output of commands into a utf-8 string.

Fixes VMAccess extension's inability to accept non latin characters as password.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (pavanrachapudy) [inline .vscode/launch.json:1]: this file needed to be there?

File: Utils/extensionutils.py [hunk 2]
```diff
@@ -200,7 +200,7 @@ def run_command_get_output(cmd, chk_err=True, log_cmd=True):
                 'CalledProcessError.  Error message is ' + str(e))
             return e.errno, str(e)
     # noinspection PyUnboundLocalVariable
-    return 0, output.decode('latin-1')
+    return 0, output.decode('utf-8')
```
Comment (pavanrachapudy) [inline Utils/extensionutils.py:203]: looks like other method(s) has this usage as well - run_send_stdin. I remember we were hitting that method for vmaccess. Also, in the current method too, it is still using latin-1? Shouldn't we handle these as well?

Comment (D1v38om83r) [reply Utils/extensionutils.py:203]: updated all output.decode to utf-8

File: .vscode/launch.json (hunk 1 above)
Comment (D1v38om83r) [reply .vscode/launch.json:1]: makes running unit tests easier

**Review by jscalev — APPROVED** [review]

**Comment by axelgMS** [comment]
thank you

**Pull request merged**